test/suite/test_cursor_random.py

```diff
@@ -71,6 +71,15 @@ def test_cursor_random(self):
         self.assertEquals(cursor.reset(), 0)
         cursor.close()
 
+    # Check that next_random fails with an empty tree, repeatedly.
+    def test_cursor_random_empty(self):
+        uri = self.type
+        self.session.create(uri, 'key_format=S,value_format=S')
+        cursor = self.session.open_cursor(uri, None, self.config)
+        for i in range(1,5):
+            self.assertTrue(cursor.next(), wiredtiger.WT_NOTFOUND)
+        cursor.close
+
```

Comment:
It looks like we should have a test that creates a tree with enough data for multiple pages, reopens the connection so we have a real tree, then truncates most / all of the tree and makes sure random lookups find data / fail (respectively).  That way we're testing {{WT_REF_DELETED}}, not just empty pages.